index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>Kieran's Kombat Academy</title>
+<title>Kieran's Kombat Academy in a branch</title>
 </head>
 <body>
```

Comment:
Get why you've opted for the h1's, as the initial task was simply to outline your layout. But do you think it makes sense to start outlining in a semantically correct way from the very start? ie. By using container elements.